test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -58,11 +33,7 @@ var server = net.createServer(function(s) {
     server.close();
   }));
 
-  // Queue up several handles, to make `process.disconnect()` wait
-  for (var i = 0; i < 100; i++)
-    send(function(err) {
-      if (err && sendcount < 3)
-        console.log(err);
-      sendcount++;
-    });
+  // Send 2 handles to make `process.disconnect()` wait
```

Comment:
Yes, what @Trott says. It was in the original version of the test. I can try to find out the reason